### ISSUE TITLE
readSVvcf is not work in findDups.R

### ISSUE THREAD

**Issue by zhqduan** [vgteam/giraffe-sv-paper#10]
Hello,

I want to construct the SV graph from several assemblies. Before starting my own works, I am testing the pipeline according to your manual, [first merge SVs from multiple datasets](https://github.com/vgteam/giraffe-sv-paper/tree/master/scripts/sv/merge-svs), and then [remap-to-dedup-merged-svs](https://github.com/vgteam/giraffe-sv-paper/tree/master/scripts/sv/remap-to-dedup-merged-svs). In the second step, when I run the findDups.R, the readSVvcf does not work with the error massage "`unused argument (vcf.object = TRUE)`". When I remove the parameter "`vcf.object=TRUE`", the readSVvcf file could work but the command "svs.gr = rowRanges(svs)" gives another error message: 
```
Error in MatrixGenerics:::.load_next_suggested_package_to_search(x) : 
  Failed to find a rowRanges() method for GRanges objects.
```
Any suggestions with this issue? Thanks in advance.

Best,
Zhongqu

**Comment by adamnovak** [comment]
I think this is an R package version problem. I think @jmonlong [got rid of the `vcf.object` argument](https://github.com/jmonlong/sveval/commit/3d123d2c5bac588110064365607bac0d689f3d07#diff-ebee64634eb57aabbe9c7c1d85438946f3ee8334e3fec1b691356d9c34709748L23-R24) when he bumped the `sveval` R package version from 1.2.2 to 2.0. So maybe you need to install `sveval` 1.2.2 for the scripts in this repo to work?

**Comment by zhoudreames** [comment]
> I think this is an R package version problem. I think @jmonlong [got rid of the `vcf.object` argument](https://github.com/jmonlong/sveval/commit/3d123d2c5bac588110064365607bac0d689f3d07#diff-ebee64634eb57aabbe9c7c1d85438946f3ee8334e3fec1b691356d9c34709748L23-R24) when he bumped the `sveval` R package version from 1.2.2 to 2.0. So maybe you need to install `sveval` 1.2.2 for the scripts in this repo to work?

Yes, I also encountered such a problem, and following your suggestion, I resolved it.

**Comment by netwon123** [comment]
sorry, i want to know how to install old version? using docker?

**Comment by adamnovak** [comment]
It looks like, to install an older version of an R package, you need to either give the full URL for that version's tarball to `install.packages`, along with `repos=NULL`, or you need to install the `remotes` package and use its `install_version` method. Both of these approaches should be explained at https://stackoverflow.com/q/17082341
